app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
it's better not to inherit from BaseException for user defined classes, because they will not be caught by regular `except Exception`, and also some system exception inherits from it. So the best practice is inherit from Exception for all exceptions you define

More details can be found here: https://stackoverflow.com/questions/27995057/why-is-it-recommended-to-derive-from-exception-instead-of-baseexception-class-in